src/components/AddTodoForm/AddTodoForm.jsx

```diff
@@ -10,19 +11,17 @@ const AddTodoForm = ({ onAddTodo }) => {
 
 	const handleAddTodo = (e) => {
 		e.preventDefault();
-		onAddTodo({ title: todoTitle, id: Date.now() });
-		setTodoTitle("");
+		if (todoTitle) {
+			onAddTodo({ title: todoTitle, id: Date.now() });
+			setTodoTitle("");
```

Comment:
Not sure if this has been resolved in future PRs, but the input isn't clearing for me whenever I submit a new text

Reply:
thanks. fixed. it slipped from me after refactoring.